lib/logger/lib/logger.ex

```diff
@@ -451,13 +459,15 @@ defmodule Logger do
   @doc """
   Adds a new translator.
   """
+  @spec add_translator({module :: atom, function :: atom}) :: :ok
   def add_translator({mod, fun} = translator) when is_atom(mod) and is_atom(fun) do
     Logger.Config.add_translator(translator)
   end
 
   @doc """
   Removes a translator.
   """
+  @spec remove_translator({module :: atom, function :: atom}) :: :ok
```

Comment:
Not until we are only on Erlang 19, since we check explicitly for `is_atom/1` so we can't use a tuple here. Sounds right?